stellargraph/core/utils.py

```diff
@@ -123,6 +123,14 @@ def chebyshev_recurrence(T_k_minus_one, T_k_minus_two, X):
     return T_k
 
 
+def PPNP_Aadj_feats_op(features, A, k=1, a=0.1, method="gcn"):
```

Comment:
I think a brief docstring would be useful here. Also the current implementation causes a warning in SciPy:
```
python3.6/site-packages/scipy/sparse/linalg/dsolve/linsolve.py:203: SparseEfficiencyWarning: spsolve is more efficient when sparse b is in the CSC matrix format
  'is in the CSC matrix format', SparseEfficiencyWarning)
```

Perhaps convert to CSC before calculating the inverse?  It is converted to the required COO format in the `SparseFullBatchNodeSequence` anyway.

Reply:
Good idea, I've chucked in a doc string. And I've converted to a numpy array instead because only `use_sparse = False` is supported for `method = 'ppnp'` now.